app/src/main/res/menu/menu_local_playlist.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <item
+        android:id="@+id/menu_item_removeWatched"
+        android:title="@string/remove_watched"
+        app:showAsAction="never"/>
```

Comment:
What about adding an icon, and making this line `app:showAsAction="ifRoom"`? The user would then be able to access the option in only one click, if there is enough space. A delete-multiple icon exists in Android Studio's library, which is the same as the Material Design icon library

Reply:
I like the idea, but If you accidentally click that button, there is no going back.
With the three dot menu, it's an extra click away, reducing the chance of accidentally removing videos you wanted to keep.
For that reason I would suggest keeping it behind the three dot menu, or at least a 'confirm action' dialog.

Reply:
Oh, there you have a point, then it is ok to keep it behind the three-dot menu